woocommerce/woocommerce-android
# Read-only variant details

This PR adds read-only product details screen. I'll fix the failing tests in a separate PR.

| Light mode | Dark mode|
| - | - |
| ![image](https://user-images.githubusercontent.com/1522856/83633206-d7143e80-a5a0-11ea-8775-d17b60fbfcbd.png) | ![image](https://user-images.githubusercontent.com/1522856/83633225-e09da680-a5a0-11ea-94fe-5c83f15e3fb2.png) |
| ![image](https://user-images.githubusercontent.com/1522856/83632809-3887dd80-a5a0-11ea-835c-07845df89523.png) | ![image](https://user-images.githubusercontent.com/1522856/83632825-3d4c9180-a5a0-11ea-86fe-9a2b26a47062.png) 
| ![image](https://user-images.githubusercontent.com/1522856/83632853-46d5f980-a5a0-11ea-8e1f-f55669e50757.png) | ![image](https://user-images.githubusercontent.com/1522856/83632869-4ccbda80-a5a0-11ea-8426-6cc6d03302d8.png) |

**To test:**

1. Open a variable product from the product list
2. Notice it shows a property group with attribute names and counts
3. Tap on Variations
4. A list of variations is loaded
5. Tap on one with an image
6. Notice a variation detail is loaded, the title is in the toolbar and it shows visibility, description (if not empty), price (including sale price or scheduled sale) and product type
7. Tap on the image
8. Notice the image detail is displayed
